index.html

```diff
@@ -0,0 +1,115 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="placeholder">
+            <img src="images/Logo Placeholder.png" alt="My Logo">
+        </div>
```

Comment:
- [ ] Excellent job enhancing the toolbar by positioning the logo on the left side :+1:. However, to further improve the meaning and structure of the code, it is advisable to enclose the logo image inside an `a` tag. This will ensure better semantic markup, make the logo clickable, and improve the user experience :+1:.